constellations/gemini-user/src/afh/afh.js

```diff
@@ -0,0 +1,126 @@
+"use strict";
+require("./afh.sass");
+import React from "react";
+import API from "../utils/api.js";
+import moment from "moment";
+
+export class AFHPage extends React.Component {
+    state = {
+        currentTab: "Math",
+        sessions: [],
+    };
+
+    componentDidMount() {
+        API.get("api/askforhelp/all").then((res) => {
+            const afh = res.data;
+            this.setState({
+                sessions: afh,
+            });
+        });
+    }
+
+    openSubject = (subjectName) => {
+        this.setState({
+            currentTab: subjectName,
+        });
+    };
+
+    render() {
+        let currentSub = this.state.sessions.filter(
+            (session) => session.subject == this.state.currentTab
+        );
+
+        let showSessions = currentSub.map((row, index) => {
+            return <AfhSessionRow key={index} row={row} />;
+        });
+
+        const subjectDisplayNames = {
+            math: "Math",
+            english: "English",
+            computer: "Computer Programming",
+        };
```

Comment:
this can be defined at the top of the file (above `export class AFHPage extends React.Component`). Treat this as a global variable